lib/Notification/Notifier.php

```diff
@@ -90,23 +91,42 @@ public function __construct(IFactory $lFactory,
 		$this->definitions = $definitions;
 	}
 
+	/**
+	 * Identifier of the notifier, only use [a-z0-9_]
+	 *
+	 * @return string
+	 * @since 17.0.0
+	 */
+	public function getID(): string {
+		return 'talk';
```

Comment:
New method in 17, replacing the `id` return of the info closure from the old registration way